internal/sinks/rpc.go

```diff
@@ -2,23 +2,52 @@ package sinks
 
 import (
 	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
 	"net/rpc"
+	"os"
+	"strings"
 
 	"github.com/cybertec-postgresql/pgwatch/v3/internal/log"
 	"github.com/cybertec-postgresql/pgwatch/v3/internal/metrics"
 )
 
-func NewRPCWriter(ctx context.Context, address string) (*RPCWriter, error) {
-	client, err := rpc.DialHTTP("tcp", address)
+func NewRPCWriter(ctx context.Context, path string) (*RPCWriter, error) {
+	address, params, found := strings.Cut(path, "?")
+	if (!found) {
+		return nil, fmt.Errorf("invalid RPC connection string, Usage: rpc://host?sslrootca=[CA_file]")
+	}
+
+	key, RootCA, found := strings.Cut(params, "=")
+	if (!found || key != "sslrootca") {
```

Comment:
We may just ignore unsupported parameters. What if server waits some additional params?